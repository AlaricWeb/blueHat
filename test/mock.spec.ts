import { expect, test, it, describe } from "vitest";
// import definRestFul, { defineModel, createData, model } from "!/restful";
import { MockRequest } from "vite-plugin-mock-dev-server";
describe("Test additions, deletions, changes and checks", () => {
  // it("Generated data", () => {
  //   const data = createData("user", { name: "@cname" });
  //   console.log(e);
  //   // expect(data).toMatchSnapshot({});
  // });
  it("bind module", () => {
    // defineModel("user", { name: "@cname" });
    // const result = definRestFul("user");
    // console.log(result);
    //@ts-ignore
  });
});
