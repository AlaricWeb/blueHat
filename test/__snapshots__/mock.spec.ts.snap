// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Test additions, deletions, changes and checks > Generated data 1`] = `
[
  {
    "created_at": "1972-01-28 07:43:20",
    "id": 1,
    "name": "夏秀英",
    "updated_at": "1978-07-30 06:08:34",
  },
  {
    "created_at": "1970-02-02 21:47:25",
    "id": 2,
    "name": "曹军",
    "updated_at": "1994-10-31 00:07:58",
  },
  {
    "created_at": "2008-02-13 20:08:08",
    "id": 3,
    "name": "龚洋",
    "updated_at": "1996-11-17 07:09:45",
  },
]
`;
